src/workloads/scale/InsertRemoveValidate.yml

```diff
@@ -0,0 +1,47 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# Demonstrate the InsertRemove actor. The InsertRemove actor is a simple actor that inserts and then
+# removes the same document from a collection in a loop. Each instance of the actor uses a different
+# document, indexed by an integer _id field. The actor records the latency of each insert and each remove.
+
+Actors:
+- Name: InsertRemoveTest
+  Type: InsertRemove
+  Threads: 99
```

Comment:
I think you want this to be 100 to compare the overhead of validate? Otherwise it's not an apples to apples comparison.